packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,170 @@
+
+### Additional requirements
+
+-   _User interaction: Do the FAST component's behaviors match the visual design spec? When they differ, which approach is preferable and why?_
+    -   Nesting Accordions: This is not covered in FAST's examples, but in their [spec](https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion/accordion.spec.md) they said that "Ideally this behavior will "just work" and no special behaviors will need to be added to enable/support this. The expectation here should be that an accordion takes content and whatever content is inside the accordion panel respects its own interaction model." This functionality was tested within FAST's example by creating a new accordion inside of one of the accordion items, and it functions. The only exception is when expand-mode="single" is used, which is further discussed in the open issues section. Using nested accordions would be preferred, as this allows more to be done with accordions and the ability to include more information without having to make one long accordion.
+    -   No Additional Requirements
+-   _Styling: Does FAST provide APIs to achieve the styling in the visual design spec?_
+    -   No Additional Requirements
+-   _Testing: Is FAST's coverage sufficient? Should we write any tests beyond Chromatic visual tests?_
+    -   No Additional Requirements
+-   _Documentation: Any requirements besides standard Storybook docs and updating the Example Client App demo?_
+    -   No Additional Requirements
+-   _Tooling: Any new tools, updates to tools, code generation, etc?_
+    -   No Additional Requirements
+-   _Accessibility: keyboard navigation/focus, form input, use with assistive technology, etc._
+    -   These are [aria's guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/) for keyboard interactions: enter or space for actions, tab and shift + tab for moving up and down, arrow keys for moving up and down between headers, and home / end (optional) for moving to the first and last accordion header, respectively. I tested all of these in the FAST component explorer, and they're functional. It is expected that these interactions will still be functional when implemented in Nimble.
+-   _Globalization: special RTL handling, swapping of icons/visuals, localization, etc._
+    -   No Additional Requirements
+-   _Performance: does the FAST component meet Nimble's performance requirements?_
+    -   No Additional Requirements
+-   _Security: Any requirements for security?_
+    -   No Additional Requirements
+
+---
+
+## Open Issues
+
+Using expand-mode="single" when nesting accordions causes issues- when trying to open nested accordions, the parent accordion closes, which makes sense as the functionality of the attribute is to only allow one accordion open at a time. Unless a reasonable fix can be found, we should advise against using this attribute with nested accordions in the docs.
```

Comment:
Does this make sense or is this a bug we need to handle?

Reply:
Seems weird for the child accordion's accordion-item to impact the parent accordion's accordion-items

Reply:
https://github.com/ni/nimble/assets/103057880/0a04c02d-1e2f-4148-9418-50c8902bb95c

Here's a video of the interactions with nested accordions in the single expand mode ^
In the first half of the video, I'm trying to close the opened accordion item through click and focusVisible, but when single expand mode is enabled, you can't close the accordion item you've opened by clicking on it, you have to click on a different accordion item. At least one accordion has to be open at all times, which could be confusing to users as the expected functionality would be that accordion items are closed until you open them and can be closed by clicking on that specific accordion item.

I look into the nested accordions in the second half of the video, where I show that once you open a child accordion, the parent accordion closes- when you open the parent accordion again, the previously selected child accordion is open, but when clicking on it, it doesn't close. Also, when opening the second child accordion, the first child accordion stays open, which goes against the functionality of single expand mode.

Regardless of the issues with the nested accordion, I don't think that single expand mode is very beneficial- having one accordion item open before users click on it, and then not allowing them to close it unless they click on a different accordion item isn't what most users would expect from the functionality of an accordion. Discussing this in the team meeting to get everyone's opinions might be helpful.

Reply:
I updated the open issue to reflect what I discussed in the comment above 👍